maintainers/maintainer-list.nix

```
@@ -6364,6 +6366,14 @@
       fingerprint = "45A9 9917 578C D629 9F5F  B5B4 C22D 4DE4 D7B3 2D19";
     }];
   };
+
+  hitsmaxft = {
+    name = "Bhe Hongtyu";
+    email = "mfthits@gmail.com";
+    github = "hitsmaxft";
+    githubId = 352727; 
```

There is a trailing whitespace causing the EditorConfig check to fail.
```suggestion
    githubId = 352727;
```

Additionally, entries in the list should not be separated by empty lines.